pytest/tests/sandbox/fast_forward.py

```diff
@@ -5,23 +5,62 @@
 import sys, time
 import pathlib
 
+from datetime import datetime as dt, timedelta
+
 sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
 
 from cluster import start_cluster
 from utils import figure_out_sandbox_binary
 
 # startup a RPC node
-BLOCKS_TO_FASTFORARD = 10000
+MIN_BLOCK_PROD_TIME = 1  # seconds
+MAX_BLOCK_PROD_TIME = 2  # seconds
+EPOCH_LENGTH = 10
+BLOCKS_TO_FASTFORWARD = 10000
 CONFIG = figure_out_sandbox_binary()
-nodes = start_cluster(1, 0, 1, CONFIG, [["epoch_length", 10]], {})
+CONFIG.update({
+    "consensus": {
+        "min_block_production_delay": {
+            "secs": MIN_BLOCK_PROD_TIME,
+            "nanos": 0,
+        },
+        "max_block_production_delay": {
+            "secs": MAX_BLOCK_PROD_TIME,
+            "nanos": 0,
+        },
+    }
+})
+
+nodes = start_cluster(1, 0, 1, CONFIG, [["epoch_length", EPOCH_LENGTH]], {})
 
 # request to fast forward
 nodes[0].json_rpc('sandbox_fast_forward', {
-    "delta_height": BLOCKS_TO_FASTFORARD,
+    "delta_height": BLOCKS_TO_FASTFORWARD,
 })
 
 # wait a little for it to fast forward
 time.sleep(3)
```

Comment:
Actually, this made me realise this should be `utils.wait_for_blocks(nodes[0], target=BLOCKS_TO_FAST_FORWARD, timeout=3)`.

Reply:
so `wait_for_blocks` doesn't work as we'd like is the thing. Fast forwarding means jumping a specific block height relative to the current block height. We won't know what the current block height is at the time of fast forwarding, and this function checks for a very specific block to be produced

Reply:
No, the function checks for a block at least equal to the target to be
produced:

        if latest.height >= target:
            return latest

so you can call it with a target equal the number of blocks being
skipped, and it’ll return as soon as the node goes past that.

Reply:
hmm weird, it threw an `AssertionError` at me before with not being able to get to `BLOCKS_TO_FAST_FORWARD` even though it went past that. Might've been out of date at that time